<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="chatArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="msgEdit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>230</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
